pkg/config/tlsconfig/config.go

```diff
@@ -177,31 +177,39 @@ func readCert(fileReader FileReader, certFile, keyFile string) (*tls.Certificate
 type ServerAuth struct {
 	Source       string     `name:"source" description:"Source of the TLS certificate (file, acme, key-vault)"`
 	FileReader   FileReader `json:"-" yaml:"-" name:"-"`
-	Certificate  string     `json:"certificate" yaml:"certificate" name:"certificate" description:"Location of TLS certificate"`
+	Certificate  string     `json:"certificate" yaml:"certificate" name:"certificate" description:"Location of TLS certificate"` //nolint:lll
 	Key          string     `json:"key" yaml:"key" name:"key" description:"Location of TLS private key"`
 	ACME         ACME       `name:"acme"`
 	KeyVault     KeyVault   `name:"key-vault"`
-	CipherSuites []string   `name:"cipher-suites" description:"DEPRECATED: List of IANA names of TLS cipher suites to use"`
+	CipherSuites []string   `name:"cipher-suites" description:"DEPRECATED: List of IANA names of TLS cipher suites to use"` //nolint:lll
 }
 
 var (
-	errInvalidTLSConfigSource = errors.DefineFailedPrecondition("tls_config_source_invalid", "invalid TLS configuration source `{source}`")
-	errEmptyTLSSource         = errors.DefineFailedPrecondition("tls_source_empty", "empty TLS source")
-	errTLSKeyVaultID          = errors.DefineFailedPrecondition("tls_key_vault_id", "invalid TLS key vault ID")
-	errInvalidCipherSuite     = errors.DefineFailedPrecondition("tls_cipher_suite_invalid", "invalid TLS cipher suite {cipher}")
+	errInvalidTLSConfigSource = errors.DefineFailedPrecondition(
```

Comment:
Ok so I've skipped `lll` on such long errors in my PRs. In the future I'll refactor this to this style instead.